### PR TITLE
Unittest meter polling

### PR DESCRIPTION
**Pull request by jameeters**
This adds a unit test for getting mamac meter data, and makes a few other changes related to this. 
This addresses #179.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,6 +70,7 @@
     "mocha": "~3.5.3",
     "mock-require": "~2.0.2",
     "redux-devtools": "~3.4.0",
+	"sinon": "~4.0.2",
```
Comment (johndiiorio) [inline package.json]: Weird tab here

File: package.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{ 
+{
   "name": "Open-Energy-Dashboard",
   "version": "0.0.0",
   "private": true,
```
Comment (johndiiorio) [inline package.json]: Unrelated to the PR, but we might want to get rid of _private: true_

File: src/server/test/db/updateMetersTests.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+		const metersToUpdate = [goodMeter, badMeter];
+
+		const dataReader = sinon.stub();
+		dataReader.withArgs(goodMeter).resolves(new Reading(
+			goodMeter.id,
+			0,
+			moment('1970-01-01 00:00:00'),
+			moment('1970-01-01 01:00:00')
+		));
+		dataReader.withArgs(badMeter).rejects(new Error('Bland error message'));
```
Comment (johndiiorio) [inline src/server/test/db/updateMetersTests.js]: Perhaps change the error message here to something more descriptive

Comment (jameeters) [reply src/server/test/db/updateMetersTests.js]: The message isn't really describing anything to begin with, but I thought the original was more descriptive.

**Pull request merged**